Basic Select/Weather-Observation-Station-8.sql

```diff
@@ -1 +1,2 @@
 select distinct city from station where left(city,1) in('a','e','i','o','u') and right(city,1) in('a','e','i','o','u') 
+
```

Comment:
Could you please delete this file and pull it again since there's no change in the file. This way it will not show a change in the PR. Thanks!